### DISCUSSION TITLE
Volume control on the RPi running MM through the integration

### DISCUSSION THREAD

**Discussion by fribse** [sindrebroch/ha-magicmirror#10] [Ideas]
**Describe the solution you'd like**
I have an 'alsa' sound card (usb sound card) in my RPi, and it would be great if I could control the volume from HA.

**Describe alternatives you've considered**
I'm trying to adjust the volume by doing some commands via SSH, it's rather difficult to get to work.

**Comment by sindrebroch** [comment]
I'll look into it, but up until now I have just used the API from https://github.com/Jopyth/MMM-Remote-Control, so this would be an addition outside of that as far as I can tell. Probably not trivial, but it would be awesome to take the integration in that direction.

**Comment by sindrebroch** [comment]
Maybe it wouldn't be that bad if another module lets us control the volume through MagicMirror's notification-system, that can communicate between modules. Maybe something like this https://github.com/Anonym-tsk/MMM-Volume? MMM-Remote-Control supports sending notifications to other modules as far as I know, but it will probably take some time to test out, as I don't have any audio connected to my mirror.

Just a theory, but it might be possible and within scope.

**Comment by fribse** [comment]
That would be very nice, just a bit concerning that the code is 3 years old on that module...

**Comment by sindrebroch** [comment]
Agreed. Will keep looking into this at some point.
